### PR TITLE
feat(loading): allow precision limiting when exporting data. Fixes #1164

### LINKED ISSUE #1164: Double precision should be configurable in exported data
Currently, when we export double data from Alchemist simulations, it does not allow us to configure the precision of the number that will be exported. In this way, the generated CSV, contains unuseful information, producing unnecessary large files.

### PR DESCRIPTION
**Pull request by DanySK**

### CODE REVIEW

File: alchemist-loading/src/main/kotlin/it/unibo/alchemist/loader/m2m/SimulationModel.kt [hunk 1]
```diff
@@ -407,12 +422,34 @@ internal object SimulationModel {
...
-                    visitBuilding(context, root)
+                    visitBuilding<Extractor<E>>(context, root)
                 } else {
                     val property = root[DocumentRoot.Export.Data.property]?.toString()
                     val filter: FilteringPolicy = root[DocumentRoot.Export.Data.valueFilter]
                         ?.let { CommonFilters.fromString(it.toString()) }
                         ?: CommonFilters.NOFILTER.filteringPolicy
+                    val precision: Int? = when (val digits = root[DocumentRoot.Export.Data.precision]) {
+                        null -> null
+                        is Int -> digits
```
Comment (cric96) [inline alchemist-loading/src/main/kotlin/it/unibo/alchemist/loader/m2m/SimulationModel.kt:433]: I dunno if it makes sense, but I would prefer something like this:
```
null, is Int -> digits
```
Why you need two separated cases?

**Review by cric96 — COMMENTED** [review]
just a stylistic doubt

File: alchemist-loading/src/main/kotlin/it/unibo/alchemist/loader/m2m/SimulationModel.kt (hunk 1 above)
Comment (DanySK) [reply alchemist-loading/src/main/kotlin/it/unibo/alchemist/loader/m2m/SimulationModel.kt:433]: Yes, nice, please PR me with that!

**Pull request merged**